src/Setup.elm

```diff
@@ -0,0 +1,166 @@
+module Setup exposing (Model)
+
+import Browser
+import Element exposing (Element, el, px, rgb255, text)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font as Font
+import Element.Input as Input
+import Html
+
+
+type alias Model =
+    { startYear : Int
+    , endYear : Int
+    , einsteinFactor : Float
+    , districtFactor : Float
+    , playOffFactor : Float
+    , offSeasonFactor : Float
+    }
+
+
+type Msg
+    = StartYear Int
+    | EndYear Int
+    | EinsteinFactor Float
+    | DistrictFactor Float
+    | PlayOffFactor Float
+    | OffSeasonFactor Float
+    | Continue
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view
+        , update = update
+        }
+
+
+init : Model
+init =
+    { startYear = 2017
+    , endYear = 2019
+    , einsteinFactor = 1
+    , districtFactor = 1
+    , playOffFactor = 1
+    , offSeasonFactor = 1
+    }
+
+
+stringToFloat : String -> Float
+stringToFloat number =
+    let
+        maybeFloat =
+            String.toFloat number
+    in
+    case maybeFloat of
+        Nothing ->
+            0
+
+        Just value ->
+            value
```

Comment:
Have you looked into [Maybe.withDefault](https://package.elm-lang.org/packages/elm/core/latest/Maybe#withDefault)?
This function isn't really needed 🙂